style.css

```diff
@@ -112,3 +112,9 @@ a {
 a:hover {
   opacity: 0.7;
 }
+
+.clearfix::after {
```

Comment:
Good job with the utility name. it makes total sense and it is easy to remember!